static/src/js/containers/GranuleFiltersContainer/__tests__/GranuleFiltersContainer.test.js

```diff
@@ -87,20 +89,12 @@ const setup = (overrideProps) => {
     handleSubmit: handleFormSubmit
   })(GranuleFiltersContainer)
 
-  // `formikBag` is the internal state of formik which we are manually going to override to match our passed props
-  let formikBag
-  // TODO update with the new way `formik` wants us to render these inline without `render` prop
+  // Rendering formik wrapper https://formik.org/docs/migrating-v2#all-render-props-have-been-deprecated-with-a-console-warning
```

Comment:
I am not sure we need this comment

Reply:
we don't have to have it its just formik warning mentioning that they at some point will deprecate the `render` prop and favor putting what is in there as a child component. I spent a while trying to get that to work but, the documentation is either not correct for use case or something similar because one of the tests here `handleReset` was not getting called. It does with the render prop but, not the child component example tried quite a few variations of this

Reply:
Maybe we file a ticket to get that updated?

Reply:
Done filed: `EDSC-4278`